src/openrct2/actions/MazePlaceTrackAction.cpp

```diff
@@ -119,7 +119,7 @@ GameActions::Result MazePlaceTrackAction::Query() const
 
     const auto& ted = GetTrackElementDescriptor(TrackElemType::Maze);
     money32 price = (((ride->GetRideTypeDescriptor().BuildCosts.TrackPrice * ted.Price) >> 16));
-    res.Cost = canBuild.Cost + price / 2 * 10;
+    res.Cost = ((canBuild.Cost + price) / 2) * 10;
```

Comment:
This differs from vanilla. For whatever reason it did `canBuild.Cost + price / 2 * 10;` It makes sense that it would do that tbh.

Reply:
Thanks for checking, this is the one change I made blindly without knowing if it's correct. Good to know it can stay.

Reply:
Does it actually make sense.

Reply:
The cost of placing a premade layout should be the same as the cost of building that layout by hand, right? Seems to me like it makes sense for them to be calculated the same way.

Reply:
@duncanspumpkin What do you think we should do? Follow vanilla or do what the other occurrences do?

Reply:
yeah keep it as per @CookiePLMonster 's design.